Makefile

```diff
@@ -1,5 +1,9 @@
 .DEFAULT_GOAL := help
 
+image_name:=alertmanager
```

Comment:
Seems like we should delete this. Is it needed as a default for something?

Reply:
I have removed the default value for them. Should we remove the declaration also?
`
image_name:=
image_tag:=latest
image_test_config:=$(image_name)/test.yaml
`